### PR TITLE
Prep for CRAN Submission

### PR DESCRIPTION
**Pull request by amy17519**
@cttobin @FCACollin @daroczig please take a look!
- Cleaned up the typos in package descriptions and functions.
- Removed the manual `scale_..` option setting per https://github.com/tidyverse/ggplot2/pull/3828/files
- Updated description 
- Fixed theme name typo -- `camoflauge` to `camouflage`

We (@cttobin and @Mikata-Project) can discuss and announce further updates on ggthemr after this :)

### CODE REVIEW

File: README.Rmd
Comment (daroczig) [inline README.Rmd:88]: See #39 -- `palettes.R` needs to be changed too?

File: DESCRIPTION
Comment (daroczig) [inline DESCRIPTION:10]: No need to include me -- I have done nothing yet :zipper_mouth_face:

Comment (amy17519) [reply DESCRIPTION]: well not exactly nothing. changing to 'ctb'

File: README.Rmd
Comment (amy17519) [reply README.Rmd:88]: 👍

**Comment by daroczig** [comment]
@amy17519 regarding writing the scales functions in the global env at https://github.com/cttobin/ggthemr/blob/0a31bb579b51b1ee4487f2957edf2227a63611d8/R/ggthemr.R#L56, ggplot2 3.3.2 might provide a more elegant solution:

> Default continuous colour scales (i.e., the options() ggplot2.continuous.colour and ggplot2.continuous.fill, which inform the type argument of scale_fill_continuous() and scale_colour_continuous()) now accept a function, which allows more control over these default continuous_scale()s (@cpsievert, #3827).

Can you please look into that?

**Comment by cttobin** [comment]
These changes look good to me. It's nice to see a proper solution for the scales is available with `options()`. That is way better than what was there before. 

One thing is I see you've corrected the spelling of "camouflage" which is obviously sensible. In the event that people already are using that theme maybe it would be safer to support both the original incorrect and corrected spellings so nobody has any broken code after upgrading. Potentially with a warning on the incorrect spelling. I'll leave that up to you to decide if it's work doing though, I wouldn't say it's critical.  

I'm happy to merge (probably "Squash and merge" unless you prefer otherwise) if you're OK with that?

**Comment by cttobin** [comment]
Or maybe we should transfer ownership first and then you can merge however you like. I'll reply to the email you Gergely sent me anyway so we can sort it out there.

File: R/ggthemr_reset.R
Comment (daroczig) [inline R/ggthemr_reset.R:10]: the code style of the package seems to use single quotes instead of double quotes -- let's stick with that?

File: R/palettes.R
Comment (daroczig) [inline R/palettes.R:330]: I think we should create a changelog / `NEWS` file and note this breaking change.

**Review by daroczig — APPROVED** [review]
@amy17519 -- I left two minor comments, but overall looks great, thanks! Let's chat if you think it makes sense addressing those two minor things, and then squash merge?

File: R/ggthemr_reset.R
Comment (amy17519) [reply R/ggthemr_reset.R:10]: good point! Sorry I was careless

**Pull request merged**
